api.py

```diff
@@ -133,15 +133,19 @@ def wait_for_job(job_id, **kwargs):
     """
     while True:
         time.sleep(0.5)
-        response = make_get_request(job_id, **kwargs)
-        response.raise_for_status()
-
-        if response.ok:
-            msg = response.json()
-            if 'response' in msg:
+        try:
+            response = make_get_request(job_id, **kwargs)
+
+            if response.ok:
+                msg = response.json()
+                if 'response' in msg:
```

Comment:
@jrmConduce should more strict checking be done here?

Reply:
When a job is still in progress, the api will return a json object containing only a `progress` field.

When a job is complete, the response will either be json containing `progress`, `response`, and `result` or plain text containing an error.  On any error, the http response won't be 200.

So I think this part is ok

Reply:
Thanks, I know we've been over that before.